db/schema.rb

```diff
@@ -769,6 +769,7 @@
     t.string "location_shelf"
     t.string "purchase_link"
     t.string "url"
+    t.jsonb "myturn_metadata", default: "{}"
```

Comment:
Looks like the MyTurn stuff from your other PR is leaking into this one

Reply:
Looks like the comment got made twice for some reason 😆

Reply:
Oh yeah, that is unintentional. Thanks for catching that!